### PR TITLE
Basic and Lambert variations examples

### PR DESCRIPTION
**Pull request by bhouston**
I also fixed a bug with Physical where reflectivity wasn't defined in some cases.

### CODE REVIEW

**Comment by bhouston** [comment]
The new examples:

http://ci.threejs.org/api/pullrequests/7396/examples/webgl_materials_lambert_variations.html
http://ci.threejs.org/api/pullrequests/7396/examples/webgl_materials_basic_variations.html

File: src/renderers/shaders/ShaderChunk/envmap_pars_fragment.glsl [hunk 1]
```diff
@@ -1,6 +1,9 @@
+#if defined( PHYSICAL ) || defined( USE_ENVMAP )
```
Comment (mrdoob) [inline src/renderers/shaders/ShaderChunk/envmap_pars_fragment.glsl]: Isn't `reflectivity` only used when `MeshPhysicalMaterial` has an `envMap`?

**Comment by bhouston** [comment]
I think you are right, I've undone that change.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**